src/components/dynamic/admin/services/judging/Toolbar.jsx

```diff
@@ -203,7 +203,12 @@ const Toolbar = ({ data, setData, judges, setJudges }) => {
             <p className="mb-0 font-semibold mx-2"># of rotations</p>
             <Button color="green" text="generate" onClick={generate} />
           </form>
-          <Button color="red" text="reset" onClick={handleReset} />
+          <Button
+            color="red"
+            text="reset"
+            onClick={handleReset}
+            disabled={data == 0}
```

Comment:
do a type check (===)

Reply:
type check (===) did not work so i did ==.

Reply:
type check means that the data is not a number, check the type of data and then compare it to that so that the type check works

Reply:
== is a jank equal and uses some assumptions, while === is way more strict